library/styleguideComponents/Wrapper/index.js

```diff
@@ -70,10 +70,63 @@ const cyanDarkTheme = {
     },
 };
 
+const pageStatus = {
+    pageName: window.location.hash.split('/')[1],
+    examplesCount: 0,
+    examplesThemes: [],
+};
+
+function registerExample() {
+    const index = pageStatus.examplesCount * 2 + 1;
+    if (!pageStatus.examplesThemes[index]) {
+        pageStatus.examplesThemes[index] = undefined;
+    }
```

Comment:
I think this is not needed since if pageStatus.examplesThemes[index] does not exists is because it is undefined already